app/models/primer.rb

```diff
@@ -5,9 +5,21 @@ class Primer < ActiveRecord::Base
   has_many :marker_assays_b, class_name: 'MarkerAssay',
              foreign_key: 'primer_b_id'
 
+  include Annotable
```

Comment:
Reason being? (I moved it to match the structure of all other model classes: (1) relationships; (2) interfaces; (3) validations; (4) custom methods.)